src/main/java/frc/robot/Constants.java

```diff
@@ -406,36 +405,34 @@ public static final class ScoringConstants {
 
         public static final double hoodPositionTolerance = 0.01;
 
-        public static final double hoodEncoderToRad = 0.0;
+        public static final double hoodEncoderToRad = (0.3947368421) * (2.0 * Math.PI);
 
-        public static final int aimLeftMotorId = 9;
-        public static final int aimRightMotorId = 10;
+        public static final int aimLeftMotorId = 16;
+        public static final int aimRightMotorId = 15;
 
         public static final int shooterLeftMotorId = 11;
         public static final int shooterRightMotorId = 12;
 
         public static final int kickerMotorId = 13;
 
-        public static final int hoodId = 14;
+        public static final int hoodId = 17;
 
-        public static final int aimEncoderPort = 0; // TODO: Change
+        public static final int aimEncoderPort = 0;
+        public static final double aimerEncoderOffset = 0.465;
 
-        public static final double aimAcceleration = 10.0;
-        public static final double aimCruiseVelocity = 10.0;
-
-        public static final double shooterAcceleration = 1;
-        public static final double shooterJerk = 1;
+        public static final double aimAcceleration = 0.1; // TODO: 15.0
+        public static final double aimCruiseVelocity = 0.1; // TODO: 15.0
```

Comment:
Weren't these supposed to be 10 each or something like 10 and 6?

Reply:
These are temporary to slow down the arm and not break it again.

Reply:
makes sense